Libraries/Storage/AsyncStorage.js

```diff
@@ -105,6 +105,22 @@ var AsyncStorage = {
   /**
    * Merges existing value with input value, assuming they are stringified json.
    * Returns a `Promise` object. Not supported by all native implementations.
+   *
+   * Example:
+   * ```JavaScript
+   *   let k1 = 'key_1'
+   *   let val_1a = JSON.stringify({'a': '123', 'b': {'b1': '234', 'b2': '234'}})
+   *   let val_1b = JSON.stringify({'b': {'b2': '555', 'b3': '555'}})
```

Comment:
I do, unless you know something I don't. I stringify there to avoid having to do it below, and I AsyncStorage only stores stringified values, else it errors. Or am I missing something?